tests/jballerina-unit-test/src/test/resources/test-src/types/xml/xml-literals-with-namespaces.bal

```diff
@@ -294,3 +294,21 @@ function testXmlInterpolationWithQuery() returns error? {
         }
     }
 }
+
+function testAddAttributeToDefaultNS() {
+    xml x1 = xml `<root xmlns="http://sample.com/wso2/c1" xmlns:ns3="http://sample.com/wso2/f"></root>`;
+    var xAttr = let var x2 = <'xml:Element>x1 in x2.getAttributes();
+    //adding attribute with default namespace
+    xAttr["{http://sample.com/wso2/c1}foo1"] = "bar1";
+    string s = x1.toString();
+    string expectedStr = "<root xmlns=\"http://sample.com/wso2/c1\" xmlns:ns3=\"http://sample.com/wso2/f\" foo1=\"bar1\"/>";
+    if (s != expectedStr) {
+        panic error("Assertion error", expected = expectedStr, found=s);
+    }
+    s = xAttr.toString();
+    expectedStr = "{\"{http://www.w3.org/2000/xmlns/}xmlns\":\"http://sample.com/wso2/c1\"," +
+    "\"{http://www.w3.org/2000/xmlns/}ns3\":\"http://sample.com/wso2/f\",\"{http://sample.com/wso2/c1}foo1\":\"bar1\"}";
+        if (s != expectedStr) {
+        panic error("Assertion error", expected = expectedStr, found=s);
+    }
```

Comment:
Shall we implement `assertEquals` method in this test file and call it? since that is the behavior we follow